### PR TITLE
remove convention controller references

### PR DESCRIPTION
**Pull request by katmutua**
## Drop Convention Controller 
In 1.5.x Convention Controller is fully deprecated and removed from TAP builds 
### Changes made 
- update examples to ensure relevance to current API in use 
- remove any references to the convention controller component
- ensure that  the controller resource is referenced as the convention service's controller to avoid confusion with the deprecated convention controller component 

### Target release
Changes targeted for the 1.5.x0-dev branch as Convention Controller will be fully deprecated in TAP 1.5.0

### CODE REVIEW

File: cartographer-conventions/creating-conventions.hbs.md [hunk 1]
```diff
@@ -73,44 +73,42 @@ the conventions are organized, grouped, and deployed is up to you and the needs
...
-do so by the second component of Cartographer Conventions, the [Convention controller](#convention-controller).
+do so by the second component of Cartographer Conventions, the [Convention service's controller](#convention-controller).
 
 ### <a id='convention-controller'></a>Convention controller
 
 The convention controller is the orchestrator of one or many convention servers deployed to the cluster. There are resources available on the `conventions.carto.run/v1aplha1` API that allow the controller to carry out its functions. These resources include:
 
   -  [ClusterPodConvention](./reference/cluster-pod-convention.hbs.md)
-      - `ClusterPodConvention` is a  resource type that allows the conventions author to register a webhook server with the controller using it's `spec.webhook` field.
+    `ClusterPodConvention` is a  resource type that allows the conventions author to register a webhook server with the controller using it's `spec.webhook` field.
```
Comment (rszumlakowski) [inline cartographer-conventions/creating-conventions.hbs.md:83]:
```suggestion
    `ClusterPodConvention` is a  resource type that allows the conventions author to register a webhook server with the controller using its `spec.webhook` field.
```
grammer;  "it's" is a contraction for "it is" and "its" is a possessive.

File: cartographer-conventions/reference/convention-resources.hbs.md [hunk 2]
```diff
@@ -1,60 +1,127 @@
...
+    - <metav1.LabelSelector>
+    webhook:
+      certificate:
+        name: sample-cert
+        namespace: sample-conventions
+      clientConfig: 
+        <admissionregistrationv1.WebhookClientConfig>
+    ```
+  A `ClusterPodConvention` can target a single or multiple workloads of different types. 
+  It is also possible to have multiple conventions being applied to a single workload
```
Comment (rszumlakowski) [inline cartographer-conventions/reference/convention-resources.hbs.md:29]:
```suggestion
  It is also possible to have multiple conventions being applied to a single workload. 
```

File: cartographer-conventions/reference/convention-resources.hbs.md [hunk 3]
```diff
@@ -1,60 +1,127 @@
...
+    webhook:
+      certificate:
+        name: sample-cert
+        namespace: sample-conventions
+      clientConfig: 
+        <admissionregistrationv1.WebhookClientConfig>
+    ```
+  A `ClusterPodConvention` can target a single or multiple workloads of different types. 
+  It is also possible to have multiple conventions being applied to a single workload 
+  How a convention is applied is solely at the discretion of the `Conventions Author`.
```
Comment (rszumlakowski) [inline cartographer-conventions/reference/convention-resources.hbs.md:30]:
```suggestion
  It is at the discretion of the "Conventions Author" how a convention is applied.
```
active voice

File: cartographer-conventions/reference/convention-resources.hbs.md [hunk 4]
```diff
@@ -1,60 +1,127 @@
...
+    conditions:
+    - <metav1.Condition>
+    template: # enriched PodTemplateSpec
+      <corev1.PodTemplateSpec>
+    ```  
+  To list out available `PodIntent` resources in your cluster, run the following `kubectl` command
+  
+  ```bash
+   # specify relevant namespace
+   kubectl get podintents.conventions.carto.run -n my-apps.
```
Comment (rszumlakowski) [inline cartographer-conventions/reference/convention-resources.hbs.md:69]:
```suggestion
   kubectl get podintents.conventions.carto.run -n my-apps
```

**Comment by HenryBorys** [comment]
Hi @katmutua Is this all set to merge?

**Pull request merged**